surveys/2023-annual-survey/questions.md

```diff
@@ -594,6 +594,21 @@ Aspects:
 >
 > This question gathers data on the communities perceptions of certain aspects of Rust at this point in time.
 
+### How do you build your Rust projects?
+
+Type: select one (optional)
+
+Aspects:
+
+- I use Cargo
+- I use some other build system
+- I combine Cargo and another build system
+- If you combine Cargo with (or just use) other build systems, which ones? (open response)
```

Comment:
One question mentioned by @joshtriplett is who drives the build system. But I'm not sure if we can express it properly. Already this last answer is quite complicated to understand. My attempt:

```
- Cargo alone
- Cargo drives another build system
- Another build system drives Cargo
- Another build system without Cargo
- What build system (apart from Cargo) do you use? (open)
```